data/json/npcs/exodii/exodii_weaponmaster_talk.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "id": "TALK_WEAPONMASTER_INTRO_1",
+    "type": "talk_topic",
+    "//": "mute by choise, and deaf most of the time (don't like to turn on the hearing cbm unless for job)",
+    "dynamic_line": "&What you see looks like the back of a mechanical spider, with at least a dozen crude but polished metal arms going from the enormously wide spine and shoulders.  Spine of a creature, meanwhile, sits in an awkward-looking chair, doing something on a wide, heavily lighted workbench.  Creature seems to be busy with something.",
```

Comment:
>polished metal arms going from -> >polished metal arms coming from 

>Spine of a creature 
I think you are referencing the mechanical spider's spine here in which case this should be
>The spine of the creature

>wide, heavily lighted 
This should probably be changed to
>wide, brightly lit
but if you wanted you could leave heavily in place of brightly. Lighted is also correct, and more popular in British English, so this one is subjective.

>Creature seems to be -> >The creature seems to be

Reply:
choise -> choice